orbit/radius.py

```diff
@@ -27,26 +27,6 @@
 # === utilities ===
 
 
-def mk_table(row_list, indentation_level=0):
```

Comment:
I was going to refactor this to remove the indentation stuff and use it for the submission table but I'll just do it later when I get there